src/wp-includes/option.php

```diff
@@ -616,6 +621,11 @@ function update_option( $option, $value, $autoload = null ) {
  * @param string      $deprecated Optional. Description. Not used anymore.
  * @param string|bool $autoload   Optional. Whether to load the option when WordPress starts up.
  *                                Default is enabled. Accepts 'no' to disable for legacy reasons.
```

Comment:
The existing note "Accepts 'no' to disable for legacy reasons." is confusing, as it may be misunderstood as in option autoloading should always be enabled, which is not correct. So I think we should remove/clarify that.

We can reuse the same explanation as on the other function:
```suggestion
 *                                Accepts 'yes'|true to enable or 'no'|false to disable. Default 'yes'.
```